docs/document/docs/guides/attachment-customization.md

```diff
@@ -0,0 +1,176 @@
+---
+id: attachment-customization
+title: Attachment customization
+sidebar_label: Attachment customization
+---
+
+## Overview
+This article assumes the following scenario: 
+1. Create a file object (blob/arrayBuffer pattern and simple object pattern) and apply it to KUC Attachment component
+2. Get the file info that user attached
+3. Validate the file type/size and show error in case it is invalid
+4. Apply KUC Attachment component files into native kintone Attachment field
+
+### Components to use
+- [Attachment](../components/desktop/attachment.md)
+- [Button](../components/desktop/button.md)
+- [Spinner](../components/desktop/spinner.md)
+
+# Completed image
+
+The completed image of the customized page is as follows:
+![attachment customize](assets/attachment_customize.gif)
+
+## JavaScript and CSS Customization
+
+When you import the UMD file of Kintone UI Component to the app, you can upload the JavaScript files by following these steps:
+
+You can see how to upload a file in the [Quick Start](../getting-started/quick-start.md).
+
+
+## What you will need to have ready
+
+Create a app that include a attachment field with id "Attachment" and a blank space filed with id "space". 
+
+### Display custom attachment area
+Display the kuc attachment component(We can directly add the files in the native kintone attachment field to the files property of kuc attachment) and two buttons:
```

Comment:
I don't know the necessity of this supplement explanation.
`(We can directly add the files in the native kintone attachment field to the files property of kuc attachment)`

```suggestion
Display the KUC Attachment component and two Button components:
```